openstack/blockstorage/v2/volumes/requests.go

```diff
@@ -0,0 +1,418 @@
+package volumes
+
+import (
+	"fmt"
+
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/pagination"
+)
+
+// CreateOptsBuilder allows extensions to add additional parameters to the
+// Create request.
+type CreateOptsBuilder interface {
+	ToVolumeCreateMap() (map[string]interface{}, error)
+}
+
+// CreateOpts contains options for creating a Volume. This object is passed to
+// the volumes.Create function. For more information about these parameters,
+// see the Volume object.
+type CreateOpts struct {
```

Comment:
Even though these parameters are not explicitly mentioned in the API docs table, they are mentioned in the example JSON request. They are also referenced in the actual API code under the `create` function:

* SourceReplica
* ConsistencyGroupID